packages/ui/src/2_molecules/Dialog/components/DialogHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, PropsWithChildren } from 'react';
+
+import { Icon } from '../../../1_atoms';
+import styles from './DialogHeader.module.css';
+
+type DialogHeaderProps = {
+  title: React.ReactNode;
+  dataActionId?: string;
+  onClose?: () => void;
+  renderTitle?: (title: React.ReactNode) => React.ReactNode;
+};
+
+export const DialogHeader: FC<DialogHeaderProps> = ({
+  title,
+  dataActionId,
+  onClose,
+  renderTitle = title => <DefaultTitle>{title}</DefaultTitle>,
+}) => {
+  return (
+    <header className={styles.container}>
+      {renderTitle(title)}
+      {onClose && (
+        <button onClick={onClose} data-action-id={dataActionId}>
+          <Icon icon="x-mark" size={12} />
+          <span className="sr-only">Close</span>
+        </button>
+      )}
+    </header>
+  );
+};
+
+export const DefaultTitle: FC<PropsWithChildren> = ({ children }) => (
```

Comment:
```suggestion 
const DefaultTitle: FC<PropsWithChildren> = ({ children }) => (
```
No need to export this.